pyproject.toml

```diff
@@ -29,7 +31,11 @@ dev = ['mypy',
        'pytest-xdist',
        'pytest-cov',
        'types-tqdm',
-       'types-Pillow']
+       'types-Pillow',
+       'types-pyyaml'
+]
 [tool.setuptools.package-data]
 "neat_ml.data" = ["*.xlsx"]
-"neat_ml"      = ["tests/baseline/*.png"]
+"neat_ml.tests.data.images" = ["*.tiff"]
+"neat_ml.tests.data.images_Processed" = ["*.tiff"]
+"neat_ml.tests.baseline" = ["*.png"]
```

Comment:
I'm a bit confused by what's going on here, given that you've re-read the diff a few times now and I'm assuming these additions are intentional?

But if they are, I don't quite get it because:

1) `find . -name "*.tiff"` returns no results in this branch. Did you actually check your code for leftovers after the migration to `pooch`? If you could do that it would save some time for your mentor here.
2) If I purge all 3 of these new lines and do a fresh re-install (`python -m pip install -v .`) after purging the old install, followed by running the testsuite in another directory, I still have everything passing, so kind of wondering what the purpose is here?